<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PolarizationScanWindow</class>
 <widget class="QMainWindow" name="PolarizationScanWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Power Scan</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
    <item>
     <widget class="QGroupBox" name="control_groupBox">
      <property name="title">
       <string>Power Scan Control</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="7" column="1">
        <widget class="QDoubleSpinBox" name="Collection_max_doubleSpinBox">
         <property name="minimum">
          <double>-360.000000000000000</double>
         </property>
         <property name="maximum">
          <double>2880.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="Excitation_max_doubleSpinBox">
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>-360.000000000000000</double>
         </property>
         <property name="maximum">
          <double>2880.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QDoubleSpinBox" name="Excitation_min_doubleSpinBox">
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>-360.000000000000000</double>
         </property>
         <property name="maximum">
          <double>2880.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Excitation Mount Max Deg</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="Fix_Excitation_Stage_checkBox">
         <property name="text">
          <string>Fix Excitation Stage</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="interrupt_pushButton">
         <property name="text">
          <string>Interrupt</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="num_datapoints_doubleSpinBox">
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="maximum">
          <double>2880.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="Fix_Collection_Stage_checkBox">
         <property name="text">
          <string>Fix Collection Stage</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="start_pushButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Number of Datapoints</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="use_shutter_checkBox">
         <property name="text">
          <string>Use Shutter</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Collection Mount Min Deg</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QDoubleSpinBox" name="Collection_min_doubleSpinBox">
         <property name="minimum">
          <double>-360.000000000000000</double>
         </property>
         <property name="maximum">
          <double>2880.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Excitation Mount Min Deg</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Collection Mount Max Deg</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="10">
        <widget class="QGroupBox" name="collect_groupBox">
         <property name="title">
          <string>Collect</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="8" column="1">
           <widget class="QDoubleSpinBox" name="IntegrationRange_max_doubleSpinBox">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>4000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QDoubleSpinBox" name="IntegrationRange_min_doubleSpinBox">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>4000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="collect_hydraharp_checkBox">
            <property name="text">
             <string>HydraHarp</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="collect_spectrum_checkBox">
            <property name="text">
             <string>Lightfield</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="hydraharp_tacq_doubleSpinBox"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>From:</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0" colspan="2">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Polarization compensation ratio (%)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="collect_apd_checkBox">
            <property name="text">
             <string>APD</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QDoubleSpinBox" name="PolarizationCompensationRatio_doubleSpinBox">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-1000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.001000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QDoubleSpinBox" name="spectrum_int_time_doubleSpinBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="apd_int_time_doubleSpinBox"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Integration range (nm)</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>To:</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0" colspan="2">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>% between S(initial) and P plane in Collection Path</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="collect_Ge_powermeter_checkBox">
            <property name="text">
             <string>Ge_Powermeter</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="collect_Si_powermeter_checkBox">
            <property name="text">
             <string>Si_Powermeter</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="plot_groupBox">
      <property name="title">
       <string>Plot</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
